src/components/TableRow/TableRow.jsx

```diff
@@ -123,14 +198,16 @@ TableRow.defaultProps = {
   colIsBadge: [],
   sectionTitle: '',
   statusCol: -1,
+  setAlertState: null,
 };
 
 TableRow.propTypes = {
-  uniqueKey: PropTypes.number,
-  data: PropTypes.arrayOf(PropTypes.node),
+  uniqueKey: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  data: PropTypes.arrayOf(PropTypes.any),
```

Comment:
🚫 **[eslint]** <[react/forbid-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/forbid-prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prop type "any" is forbidden

Reply:
table row is too general for proper prop typing of data